airflow/www/static/js/base.js

```diff
@@ -38,6 +38,24 @@ export function escapeHtml(text) {
 
 window.escapeHtml = escapeHtml;
 
+// Handle i18n localization of date + time (D MMMM YYYY HH:mm)
```

Comment:
Does this include seconds or not? (i.e. is the comment wrong, or are seconds not included in output)